hls4ml/utils/config.py

```diff
@@ -32,12 +32,24 @@ def _get_precision_from_quantizer(quantizer):
             quantizer['class_name'] = quantizer_obj.__class__.__name__
             quantizer['config'] = quantizer_obj.get_config()
         # Some activations are just functions
-        else: 
+        else:
             quantizer['class_name'] = quantizer_obj.__name__
 
-    supported_quantizers = ['quantized_bits', 'quantized_relu', 'quantized_tanh', 'quantized_po2', 'quantized_relu_po2']
+    supported_quantizers = ['quantized_bits', 'quantized_relu', 'quantized_tanh',
+                            'quantized_sigmoid', 'quantized_po2', 'quantized_relu_po2']
     signed = True
-    if quantizer['class_name'] in supported_quantizers:
+    rnd = "AP_RND_CONV"
+    overflow = "AP_SAT"
+
+    if quantizer['class_name'] == 'quantized_tanh':
+        overflow = "AP_SAT_SYM" if quantizer['config']['symmetric'] else "AP_SAT"
+        bits = int(quantizer['config']['bits'])
+        integer = 1
+    elif quantizer['class_name'] == 'quantized_sigmoid':
+        bits = int(quantizer['config']['bits'])
+        integer = 0
+        signed = False
```

Comment:
Maybe move these into the `if quantizer['class_name'] in supported_quantizers:` statement, as an `elif` after `if quantizer['class_name'] == 'quantized_relu'` - just to improve readability so all the special cases after the logic that applies to them all.

Reply:
Done